code/modules/clothing/spacesuits/hardsuit.dm

```diff
@@ -103,7 +103,25 @@
 	var/helmettype = /obj/item/clothing/head/helmet/space/hardsuit
 	var/obj/item/tank/jetpack/suit/jetpack = null
 
+	//Modular start
+	var/allowed_modules
+
+	var/initial_modules = list()
+
+	var/datum/modular_hardsuit/mod_suit
```

Comment:
What's the point of using a datum here, just have everything on the suit itself

Reply:
you're disgusting

Reply:
disgusting but correct